openedx/core/djangoapps/user_authn/views/login.py

```diff
@@ -577,7 +577,7 @@ def login_user(request, api_version='v1'):
         if error_code:
             set_custom_attribute('login_error_code', error_code)
 
-        response_content['email'] = request.POST.get('email', None)
+        response_content['email'] = request.POST.get('email' if api_version == API_V1 else 'email_or_username', None)
```

Comment:
use a separate variable to figure-out the arg name and use that var to retrieve `post param`.